vignettes/getting-started-part-I.Rmd

```diff
@@ -0,0 +1,220 @@
+We process reported data into the format required for `epinowcast` and return it in a `data.table`. At this stage, we need to specify a grouping (i.e age, location) if any. 
+
+```{r}
+pobs <- enw_preprocess_data(retro_nat_germany, max_delay = 40)
+pobs
+```
+
+The returned output is in the form of a `data.table` with metadata stored as variables. It can be useful to check this output before specifying the model, just to make sure everything is as expected.
+
+# Model specification
+
+The `epinowcast` package is designed to provide users with a flexible and customizable modelling framework. The package comes equipped with several modules that users can utilize to construct models, and also allows users to create their own modules. These ensures that models can be tailored to the user's specific data and context.
+
+## Default nowcasting model
+
+The default nowcasting model in `epinowcast` consists of three modules: 
+
+- A process (expectation) module that models the expected counts by date of reference (`reference_date`)
+- A parametric reference reporting model which models the reporting delay distribution from the date of reference
+- A non-parametric reporting model which models differences in the reporting delay distribution by date of report (`report_date`), for example, day-of-the-week effects on the reporting delay.
+
+In the following sections, we specify simple models for each of these modules. The appropriateness of these specifications will vary depending on your context. See our vignettes for further details on model specification and examples of more complex models.
+
+## Process model
+
+A commonly used process model in nowcasting is to model the expected counts by date of reference via a geometric random walk as this acts as a minimally informed smoothing prior and thus gives a lot of weight to the observed data. This is the default process model in `epinowcast`. Users may also specify this model for themselves using the enw_expectation() function.
+
+```{r}
+expectation_module <- enw_expectation(
+  ~ 0 + (1 | day), data = pobs
+)
+```
+
+Here, `day` refers to the number of days from the start of the data.
+
+As the underlying process model is an exponential growth rate model ($C_t = C_{t-1} \exp^{r_t}$), specifying a random effect (i.e. `(1 | day)`) on the growth rate is equivalent to a geometric random walk on expected counts by reference date.
+
+## Reporting model by reference date
+
+Our baseline assumption for the reporting delay is that it is log-normally distributed, and static over time and strata. We can specify this model using the `enw_reference()` function,
+
+```{r}
+reference_module <- enw_reference(~1, distribution = "lognormal", data = pobs)
```

Comment:
```suggestion
reference_module <- enw_reference(parametric  = ~1, distribution = "lognormal", data = pobs)
```

Reply:
Side note: What's the preferred style? First argument after the opening bracket, then others below? All on the same line like here, or first argument starts on a new line after the opening bracket (line in previous command above)?

Reply:
I've linked an issue.